commands/config_test.go

```diff
@@ -0,0 +1,382 @@
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlUnitTestSuite) TestConfigGetCmd() {
+	s.Run("Get a string config value for a given key", func() {
+		printer.Clean()
+		args := []string{"SqlSettings.DriverName"}
+		outputConfig := &model.Config{}
+		outputConfig.SetDefaults()
+
+		s.client.
+			EXPECT().
+			GetConfig().
+			Return(outputConfig, &model.Response{Error: nil}).
+			Times(1)
+
+		err := configGetCmdF(s.client, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(*(printer.GetLines()[0].(*string)), "mysql")
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Get an int config value for a given key", func() {
+		printer.Clean()
+		args := []string{"SqlSettings.MaxIdleConns"}
+		outputConfig := &model.Config{}
+		outputConfig.SetDefaults()
+
+		s.client.
+			EXPECT().
+			GetConfig().
+			Return(outputConfig, &model.Response{Error: nil}).
+			Times(1)
+
+		err := configGetCmdF(s.client, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(*(printer.GetLines()[0].(*int)), 20)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Get a boolean config value for a given key", func() {
+		printer.Clean()
+		args := []string{"SqlSettings.Trace"}
+		outputConfig := &model.Config{}
+		outputConfig.SetDefaults()
+
+		s.client.
+			EXPECT().
+			GetConfig().
+			Return(outputConfig, &model.Response{Error: nil}).
+			Times(1)
+
+		err := configGetCmdF(s.client, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(*(printer.GetLines()[0].(*bool)), false)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Get a slice of string config value for a given key", func() {
+		printer.Clean()
+		args := []string{"SqlSettings.DataSourceReplicas"}
+		outputConfig := &model.Config{}
+		outputConfig.SetDefaults()
+
+		s.client.
+			EXPECT().
+			GetConfig().
+			Return(outputConfig, &model.Response{Error: nil}).
+			Times(1)
+
+		err := configGetCmdF(s.client, &cobra.Command{}, args)
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		s.Require().Equal(printer.GetLines()[0].([]string), []string{})
```

Comment:
We should be able to remove the casting here:

```suggestion
		s.Require().Equal(printer.GetLines()[0], []string{})
```